graphql.go

```diff
@@ -42,6 +42,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const errorNon200Template = "graphql: server returned a non-200 status code: %v"
```

Comment:
This feels like an unnecessary constant to me. Let's just duplicate the string for now?

Reply:
(Great PR btw)

Reply:
thanks @matryer, I pulled the constant and duplicated the string.